style.css

```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap");
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: sans-serif;
+  font-weight: 400px;
+  background: #e5e5e5;
+}
+
+section {
+  width: 375px;
+  height: 718px;
+  background: #fff;
+}
+

```

Comment:
Kindly add a background image as you have it in the template 😊.
![no-image](https://user-images.githubusercontent.com/73854999/198389513-8cbc6e45-fd59-45d5-8f61-8142c38acfa4.PNG)
This is the template👇
![temp](https://user-images.githubusercontent.com/73854999/198390648-6cfaf95c-f43d-4742-82f4-8cba28946728.PNG)